src/components/Capture/index.js

```diff
@@ -222,8 +222,3 @@ const CaptureScreen = ({method, useCapture, hasCaptured, ...other})=> (
     }
   </div>
 )
-
```

Comment:
not problematic to leave this on, considering that the component can still be used in isolation as it stands now, might be argued that we don't want to do that though.